data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -153,6 +153,32 @@
     "Percent of individuals < 100% Federal Poverty Line in 2010"
 )
 
+# RSEI Aggregated Micro-data
+EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD = (
+    "Number of facilities affecting the tract"
+)
+EPA_RSEI_NUMBER_RELEASES_OUTPUT_FIELD = "Number of releases affecting the tract"
+EPA_RSEI_NUMBER_CHEMICALS_OUTPUT_FIELD = (
+    "Number of chemicals affecting the tract"
+)
+EPA_RSEI_AVERAGE_TOXICITY_OUTPUT_FIELD = (
+    "Average toxicity-weighted concentration of the cells in the tract"
+)
+EPA_RSEI_SCORE_OUTPUT_FIELD = (
+    "Sum of the risk-related scores of the cells in the tract"
```

Comment:
Similar comment: "risk-related scores" is too vague. Please rename this field something like `"RSEI Risk Score"`.

Reply:
I am following the documentation from the EPA, but will make the change.